src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) ?? tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={tab.id === activeTab.id ? 'is-active' : ''}
+              data-cy="Tab"
+            >
+              <a
+                onClick={
+                  tab.id !== activeTab.id
+                    ? () => onTabSelected(tab.id)
+                    : undefined
+                }
```

Comment:
Better to create separate fucntion for this

`function handleTabSelect(tabId) {
  if(tabId === activeTab.id) return;
  onTabSelected(tab.id);
}`

And use this function here

Reply:
as for me, this method is more obvious